DogeNetWebClient/src/app/core/services/auth.service.ts

```diff
@@ -24,6 +18,9 @@ export class AuthService {
     this.oauthService.configure(authCodeFlowConfig);
     this.oauthService.loadDiscoveryDocumentAndTryLogin();
     this.oauthService.setupAutomaticSilentRefresh();
+    setTimeout(() => {
```

Comment:
Why do we need a timeout here? If you are waiting for methods above, you need to wait until it ends not for some const time period.